File: Lecture06_MaterialDesign_Binding/AfterWorkshop_Java/app/src/main/java/com/lavima/afterworkshop_java/MainActivity.java
package com.lavima.afterworkshop_java;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.app.AppCompatDelegate;

import android.content.res.Configuration;
import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        // This does not work when uiMode config changes are enabled in the manifest
        //AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES);
    }

    @Override
    public void onConfigurationChanged(@NonNull Configuration newConfig) {
        super.onConfigurationChanged(newConfig);
        Log.d("MainActivity", "onConfigurationChanged");
        int currentNightMode = newConfig.uiMode & Configuration.UI_MODE_NIGHT_MASK;
        switch (currentNightMode) {
            case Configuration.UI_MODE_NIGHT_NO:
                Log.d("MainActivity", "Light theme");
                break;
            case Configuration.UI_MODE_NIGHT_YES:
                Log.d("MainActivity", "Dark theme");
                break;
        }
    }
}

File: Lecture08_ViewModel/Before_Kotlin_ViewModel/app/src/main/java/no/hiof/larseknu/playingwithviewmodel/MainActivity.kt
package no.hiof.larseknu.playingwithviewmodel

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.lifecycle.ViewModel
import androidx.lifecycle.ViewModelProvider
import androidx.lifecycle.ViewModelProviders
import no.hiof.larseknu.playingwithviewmodel.model.Movie

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


    }
}
